src/components/Tabs.jsx

```diff
@@ -42,25 +63,44 @@ class Tabs extends React.Component {
         // ensures that single or no child components don't throw
         const children = React.Children.toArray(this.props.children)
 
+        let activeTab = null
+        // in case of invalid activeTab value, render first tab instead
+        if (children.length > 0) {
+            activeTab = this.valueToTab[this.state.activeTabValue] || children[0]
+        }
+
+        const cls = classNames(
+            "reactist",
+            "tabs",
+            this.props.spreadLayout ? "tabs-spreadlayout" : null
+        )
+
         return (
-            <div className='reactist tabs'>
+            <div className={cls}>
                 <div className='reactist tabs__header'>
                     { this._renderTabLinks(children) }
                 </div>
                 <div className='reactist tabs__body'>
-                    { children[this.state.activeTab] }
+                    { activeTab }
                 </div>
             </div>
         )
     }
 }
 Tabs.displayName = 'Tabs'
-Tabs.defaultProps = {
-    defaultTab: 0
-}
+
 Tabs.propTypes = {
-    /** Index of the tab that should be selected on initial render. */
-    defaultTab: PropTypes.number
+    /** selects the tab whose value prop matches this prop */
+    defaultTab: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
+    ]),
+    spreadLayout: PropTypes.bool,
```

Comment:
Could you add a jsdoc comment so a small description appears in the storybook a la `Whether the tabs should take all available space and distribute it evenly or use the minimum required`